Editor/Generators/ReactAction/ReactAction.Generator.tt

```diff
@@ -0,0 +1,124 @@
+<#@ template language="C#" #>
+<#@ output extension=".cs" #>
+<#@ import namespace="System" #>
+<#@ import namespace="System.IO" #>
+<#@ import namespace="System.Text" #>
+<#@ import namespace="System.Collections.Generic" #>
+<#@ import namespace="System.Linq" #>
+<#@ import namespace="System.Xml.Linq" #>
+<#@ import namespace="ReactUnity.Editor.Generators"#>
+<#=((CommonGenerators)Session["Common"]).AutoGeneratedWarning#>>
+using System;
+using ReactUnity.Helpers;
+
+namespace ReactUnity.Helpers
+{
+<#
+  for (int i = 0; i < 10; i++)
+  {#>
+  /// <summary>
+  ///   A wrapper for the <see cref="ReactUnity.Helpers.Callback" /> class to provide convienient access to the
+  ///   delegates and events in ReactUnity from both JavaScript and TypeScript as well as C# as normal events.
+  /// </summary>
+	/// <example>
+	///   <code language="csharp">
+	///     var callback = new ReactAction&lt;string&gt;();
+	///     callback += (str) => Console.WriteLine(str);
+	///     callback.Invoke("It works! From C#");
+	///     callback -= (str) => Console.WriteLine(str);
```

Comment:
This example is wrong. Subtracted delegate is not the same reference as added delegate.

Reply:
You are right, I didn't look closely what copilot suggested